src/AutoBuild.jl

```diff
@@ -212,6 +220,15 @@ function build_tarballs(ARGS, src_name, src_version, sources, script,
         end
     end
 
+    # Modify script for debugging
+    if debug
+        if debug_mode == "begin"
+            script = "false\n" * script
```

Comment:
In this case it could simply be
```suggestion
            script = "false"
```
I don't have a strong opinion on either way

Reply:
Let's keep the script, we may eventually want to add a "continue" option.